client/dist/index.html

```diff
@@ -1,9 +1,13 @@
 <!doctype html>
 <html>
-  <head>
-  </head>
-  <body>
-    <div id="app"></div>
-    <script src="bundle.js"></script>
-  </body>
+
+<head>
+  <link rel="stylesheet" href="style.css">
```

Comment:
Although not mentioned, you should add a meta tag specifying the character set. If you go to the console you'll get a warning about this. Some characters/icons could render weirdly

Reply:
Added the charset.